src/components/DateRangePicker/SelectionPane.js

```diff
@@ -0,0 +1,128 @@
+
+    return (
+      <div style={styles.container}>
+        <div style={{ borderBottom: isLargeOrMediumWindowSize ? 'none' : '1px solid ' + StyleConstants.Colors.FOG }}>
+          <label style={styles.boxLabel}>From</label>
+          <div onClick={() => this._handleDateBoxClick(selectedStartDate, 'from')} style={Object.assign({}, styles.dateSelectBox, this.props.selectedBox === 'from' ? styles.selectedDateSelectBox : null)}>{selectedStartDate ? moment.unix(selectedStartDate).format('MMM D, YYYY') : 'Select Start Date'}</div>
+
+          <label style={styles.boxLabel}>To</label>
+          <div onClick={() => this._handleDateBoxClick(selectedEndDate, 'to')} style={Object.assign({}, styles.dateSelectBox, this.props.selectedBox === 'to' ? styles.selectedDateSelectBox : null)}>{selectedEndDate ? moment.unix(selectedEndDate).format('MMM D, YYYY') : 'Select End Date'}</div>
+        </div>
+        <div>
+          <div style={Object.assign({}, styles.defaultRangesTitle, { color: this.props.primaryColor })}>
+            Select a Range
+          </div>
+          <DefaultRanges {...this.props} styles={styles} />
+        </div>
+      </div>
+    );
+  }
+
+  styles = () => {
+    const { isLargeOrMediumWindowSize } = this.props;
+
+    return {
+      container: {
+        display: 'flex',
+        flexDirection: 'column',
+        height: '100%',
+        borderRight: isLargeOrMediumWindowSize ? '1px solid ' + StyleConstants.Colors.FOG : 'none',
+        padding: StyleConstants.Spacing.MEDIUM
+      },
+      calendarHeaderNav: {
+        width: 35,
+        cursor: 'pointer'
+      },
+
+      boxLabel: {
+        fontFamily: StyleConstants.FontFamily,
+        fontSize: StyleConstants.FontSizes.MEDIUM,
+        color: StyleConstants.Colors.CHARCOAL
+      },
+      dateSelectBox: {
+        borderColor: StyleConstants.Colors.FOG,
+        borderRadius: 3,
+        borderStyle: 'solid',
+        borderWidth: 1,
+        boxSizing: 'border-box',
+        cursor: 'pointer',
+        fontFamily: StyleConstants.FontFamily,
+        fontSize: StyleConstants.FontSizes.MEDIUM,
+        marginBottom: StyleConstants.Spacing.SMALL,
+        marginTop: StyleConstants.Spacing.XSMALL,
+        padding: '10px 15px'
+      },
+      selectedDateSelectBox: {
+        borderColor: this.props.primaryColor,
+        cursor: 'pointer',
+        color: this.props.primaryColor
+      },
+
+      //Default Ranges
+      defaultRangesTitle: {
+        color: StyleConstants.Colors.PRIMARY,
+        fontFamily: StyleConstants.Fonts.SEMIBOLD,
+        fontSize: StyleConstants.FontSizes.SMALL,
+        padding: `${StyleConstants.Spacing.LARGE}px 0px ${StyleConstants.Spacing.SMALL}px ${StyleConstants.Spacing.LARGE}px`
+      },
+      rangeOptions: {
+        boxSizing: 'border-box',
+        color: StyleConstants.Colors.CHARCOAL,
+        display: 'flex',
+        flexWrap: 'wrap',
```

Comment:
the component that the `rangeOptions` style is applied to maps over the ranges. if i did it in rows wouldn't i need to set a new element for every two elements in the ranges? this is nice because no matter the number of items in the range it works nicely: 

<img width="625" alt="screen shot 2017-07-18 at 7 10 42 pm" src="https://user-images.githubusercontent.com/13579578/28346235-dea9261a-6bec-11e7-9f67-1cccf3d0c397.png">


if you're thinking of something i'm not though lmk